Part2_Mapplots/ex2_0_intro_scatter.ipynb

```diff
@@ -171,7 +193,7 @@
    "source": [
     "#### Note\n",
     "\n",
-    " > It's difficult to get a nice layout with `<GeoAxesSubplot>`. This is discussed at length in [Exercise 2.7](ex2_7_subplots.ipynb)."
+    " > Because map plots have a fixes aspect ratio it's difficult to get a nice layout with `<GeoAxesSubplot>`. This is discussed in [Exercise 2.7](ex2_7_subplots.ipynb)."
```

Comment:
```suggestion
    " > Because map plots have a fixed aspect ratio it's difficult to get a nice layout with `<GeoAxesSubplot>`. This is discussed in [Exercise 2.7](ex2_7_subplots.ipynb)."
```